### PR TITLE
Create a simple implementation of fee-less wires

### PR DESCRIPTION
**Pull request by sampoder**
Needs testing, progress towards https://github.com/hackclub/hcb/issues/8104.

### CODE REVIEW

File: app/models/event.rb [hunk 1]
```diff
@@ -745,6 +745,14 @@ def short_name(length: 16)
     self[:short_name] || name[0...length]
   end
 
+  monetize :minimumn_wire_amount_cents
+
+  def minimumn_wire_amount_cents
+    return 1 if settled_incoming_balance_cents(start_date: 1.year.ago) + fronted_incoming_balance_v2_cents(start_date: 1.year.ago) > 50_000
```
Comment (sampoder) [inline app/models/event.rb]:
```suggestion
    return 1 if canonical_transactions.where("amount_cents > 0").where("date >= ?", 1.year.ago).sum(:amount_cents) > 50_000_00
```

File: app/models/event.rb [hunk 2]
```diff
@@ -745,6 +745,14 @@ def short_name(length: 16)
     self[:short_name] || name[0...length]
   end
 
+  monetize :minimumn_wire_amount_cents
+
+  def minimumn_wire_amount_cents
+    return 1 if canonical_transactions.where("amount_cents > 0").where("date >= ?", 1.year.ago).sum(:amount_cents) > 50_000_00
```
Comment (garyhtou) [inline app/models/event.rb]:
```suggestion
    return 100 if canonical_transactions.where("amount_cents > 0").where("date >= ?", 1.year.ago).sum(:amount_cents) > 50_000_00
```

File: app/views/wires/new.html.erb [hunk 3]
```diff
@@ -10,10 +10,16 @@
...
+<% callout_footer += " (admins can bypass this)" if current_user.admin? %>
+
 <%= render "callout",
     type: "info",
-    title: "Important information about wire transfers" do %>
-    <span class="font-bold">Each wire transfer incurs a <%= render_money Wire::ESTIMATED_FEE_CENTS_USD %> flat fee</span>, this will be added on to the amount entered below and is passed on from our partner bank. Additionally, <span class="font-bold">currency exchange rates may fluctuate</span> between a wire being requested and a wire being sent.
+    title: "Important information about wire transfers",
+    footer: callout_footer do %>
+    Unfortunately, wires are expensive to send. Our partner bank charges us $25 per wire; therefore, to keep HCB sustainable without passing this fee onto you,
+    <span class="font-bold">each wire sent must be at least $500</span> (after conversion to USD). Consider a PayPal transfer for smaller amounts or reach out to <%= mail_to "hcb@hackclub.com" %>.
```
Comment (garyhtou) [inline app/views/wires/new.html.erb]: Link to PayPal transfer

Comment (sampoder) [inline app/views/wires/new.html.erb]:
```suggestion
    <span class="font-bold">each wire sent must be at least $500</span> (after conversion to USD). Consider a <%= link_to "PayPal transfer", new_event_paypal_transfer_path(@event) %> for smaller amounts or reach out to <%= mail_to "hcb@hackclub.com" %>.
```

**Pull request merged**